pkgs/by-name/po/poptop/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  ...
```

Comment:
```suggestion
```

no need for `...`, since `callPackage` (which is what's used for most packages and also `pkgs/by-name`) auto-detects what arguments are listed inside the top `{}:`, and only provides those values.